src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -29,12 +30,14 @@
 
 public class HelperUtils {
 
+  private static final Pattern PONUMBER_PATTERN = Pattern.compile("^[a-zA-Z0-9]{5,16}$");
   public static final String DEFAULT_POLINE_LIMIT = "500";
   public static final String OKAPI_URL = "X-Okapi-Url";
   public static final String PO_LINES_LIMIT_PROPERTY = "poLines-limit";
   public static final String URL_WITH_LANG_PARAM = "%s?lang=%s";
   public static final String GET_ALL_POLINES_QUERY_WITH_LIMIT = resourcesPath(PO_LINES)+"?limit=%s&query=purchase_order_id==%s&lang=%s";
-  public static final String GET_ALL_PURCHASE_ORDER_QUERY = resourceByIdPath(PURCHASE_ORDER)+URL_WITH_LANG_PARAM;
+  public static final String GET_PURCHASE_ORDER_BYID = resourceByIdPath(PURCHASE_ORDER)+URL_WITH_LANG_PARAM;
```

Comment:
Where is `PURCHASE_ORDER` being defined or how will it get its value?

Reply:
It is defined in the SubObjects.java. It is accessed via a static import